src/ripple/app/rdb/backend/FlatmapDatabase.h

```diff
@@ -67,7 +65,7 @@ class FlatmapDatabase : public SQLiteDatabase
 
 public:
     FlatmapDatabase(Application& app, Config const& config, JobQueue& jobQueue)
-        : app_(app), config_(config), jobQueue_(jobQueue)
```

Comment:
why are these types of changes in the conan pr?

Reply:
Separate PR merged before this sounds good!?

Reply:
yeah I just don't want code changes in a build system pr... it's too much mixing. if we need to revert build system pr we shouldn't be rolling back code changes at the same time, it's messy